packages/excalidraw/element/resizeElements.ts

```diff
@@ -54,6 +61,9 @@ import { LinearElementEditor } from "./linearElementEditor";
 import { isInGroup } from "../groups";
 import { mutateElbowArrow } from "./routing";
 
+let alreadyFlippedX = false;
+let alreadyFlippedY = false;
```

Comment:
@dwelle if you have a better recommendation I'm open to it. Didn't want to wire through an app state update.